calico.go

```diff
@@ -284,6 +285,28 @@ func cmdDel(args *skel.CmdArgs) error {
 	fmt.Fprintf(os.Stderr, "Calico CNI releasing IP address\n")
 	logger.WithFields(log.Fields{"paths": os.Getenv("CNI_PATH"),
 		"type": conf.IPAM.Type}).Debug("Looking for IPAM plugin in paths")
+
+	// We need to replace "usePodCidr" with a valid, but dummy podCidr string with "host-local" IPAM.
+	if conf.IPAM.Type == "host-local" && strings.EqualFold(conf.IPAM.Subnet, "usePodCidr") {
+
+		// Use a dummy CIDR from IETF example IP range.
```

Comment:
We probably shouldn't use the example range now that I think about it, given they are meant for examples.

Does "0.0.0.0/0" work?